osquery/tables/specs/utility/osquery_schedule.table

```diff
@@ -1,9 +1,15 @@
 table_name("osquery_schedule")
 description("Information about the current queries that are scheduled in osquery.")
 schema([
-    Column("interval", INTEGER, "The interval in seconds to run this query, not an exact interval."),
-    Column("name", TEXT, "The given name for this query."),
-    Column("query", TEXT, "The exact query to run."),
+    Column("name", TEXT, "The given name for this query"),
+    Column("query", TEXT, "The exact query to run"),
+    Column("interval", INTEGER, "The interval in seconds to run this query, not an exact interval"),
+    Column("executions", INTEGER, "Number of times the query was executed"),
+    Column("bytes", INTEGER, "Total number of bytes generated by the query"),
```

Comment:
@jacknagz, what do you think of these column names and descriptions? Do these additions cover the features/data you'd like to see?